docs/src/tutorial.md

```diff
@@ -60,22 +78,26 @@ the semi-colon trailing `plot`.  This automatically renders the image to
 your default multimedia display, typically an internet browser.  No need
 to capture the output argument in this case.
 
+```julia
+plot(iris, x=:SepalLength, y=:SepalWidth)
 ```
-plot(iris, x=:SepalLength, y=:SepalWidth, Geom.point)
-```
+
+Note that there is also no need to specify `Geom.point`, as that will be
+supplied automatically if no other geometries are given.
```

Comment:
Should we be recommending this behavior? IMO it's better to be explicit about what we're plotting.

Reply:
if we ever removed Geom.point as a default, then we should make this change.  otherwise best to document it i think.

Reply:
Perhaps we could say something along the lines of:

> In cases where no explicit geometry is provided, the plot will use `Geom.point` by default.

This way, it doesn't sound like we're recommending this behavior, but it does document that this is the default.